components/resources/brave_new_tab_page_strings.grdp

```diff
@@ -1,4 +1,48 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <grit-part>
-  <!-- Add strings for Brave new tab page here. -->
+
+  <message name="IDS_NEW_TAB_BACKGROUND_SETTINGS_TITLE" desc="Title for background image settings">
+    Background Image
+  </message>
+
+  <message name="IDS_NEW_TAB_BRAVE_BACKGROUND_LABEL" desc="Label for Brave background option">
+    Brave backgrounds
+  </message>
+
+  <message name="IDS_NEW_TAB_CUSTOM_BACKGROUND_LABEL" desc="Label for custom background option">
+    Use your own
+  </message>
+
+  <message name="IDS_NEW_TAB_GRADIENT_BACKGROUND_LABEL" desc="Gradient background colors title">
+    Gradients
+  </message>
+
+  <message name="IDS_NEW_TAB_PHOTO_CREDITS_TEXT" desc="Photographer credits for Brave background images">
+    Photo by <ph name="AUTHOR">$1</ph>
+  </message>
+
+  <message name="IDS_NEW_TAB_RANDOMIZE_BACKGROUND_LABEL" desc="Label for random toggle button">
+    Refresh on every new tab
+  </message>
+
+  <message name="IDS_NEW_TAB_SETTINGS_TITLE" desc="Title for settings dialog">
+    Customize Dashboard
```

Comment:
Some of these are capital case - some are just sentence case. Maybe just check that - should it be consistent?